build_tools/ci/cpu_comparison/run.py

```diff
@@ -1023,6 +1028,15 @@ def all_tests(
         default="",
     )
 
+    parser.add_argument(
+        "--device-hal",
+        default="xrt-lite",
+        const="xrt-lite",
+        nargs="?",
```

Comment:
What are "const" and "nargs" used for? Also what's the difference between "xrt" and "xrt-lite"? When should we choose "xrt"/"xrt-lite"?

Reply:
re `const` and `nargs`: https://stackoverflow.com/a/40324463

re `xrt` vs. `xrt-lite`: this PR adds the `xrt-lite` HAL (the previous/current/original HAL is the `xrt` HAL). If you're on Linux you should choose `xrt-lite` and if you're on Windows, for now, you should choose `xrt`.